definitions/3.0.0/components.json

```diff
@@ -68,6 +68,21 @@
         }
       }
     },
+    "operations": {
+      "type": "object",
+      "patternProperties": {
+        "^[\\w\\d\\.\\-_]+$": {
+          "oneOf": [
+            {
+              "$ref": "http://asyncapi.com/definitions/3.0.0/Reference.json"
+            },
+            {
+              "$ref": "http://asyncapi.com/definitions/3.0.0/operation.json"
```

Comment:
I was a bit confused here because the spec references `Operation Item Object`. 

![image](https://github.com/asyncapi/spec-json-schemas/assets/193286/6a2be416-98f9-421e-8dd2-d4c7fc0501f1)

But it looks like it's an editorial error which I've addressed in: https://github.com/asyncapi/spec/pull/988